changelog/changelog_3.0.0-4.0.0.txt

```diff
@@ -1,3 +1,21 @@
+06.08.2024
+Description
+  - Intergration the Sync Component
```

Comment:
We can try a bit harder here :D Explaining what it is and what the required tasks to integrate it are.